components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -606,13 +606,26 @@ struct HardwareInfo {
   string device_id;
 };
 
+enum AccountKind {
+  kDerived,
```

Comment:
nit: No "k" prefix is used in other enums

Reply:
k prefix is preferred by style guide. 
https://chromium.googlesource.com/chromium/src/+/lkgr/mojo/public/tools/bindings/README.md#enumeration-types
Let's eventually fix other enums in this file ;)